recipes/libevent/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Needed to ensure our OpenSSL and
+# not the system one is used on OS X.
+export LIBRARY_PATH="${PREFIX}/lib"
+
+export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+
+# Set the fallback library environment variable.
+if [[ `uname` == 'Darwin' ]];
```

Comment:
Both [work]( http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_03_04.html#sect_03_04_04 ) (albeit slightly differently). That being said, I don't have a preference. So went ahead and changed it.